### ISSUE TITLE
Which version is "Default"?

### ISSUE THREAD

**Issue by svick** [ashmind/SharpLab#240]
When I'm using SharpLab to investigate the behavior of Roslyn, it's very useful to know exactly which version of Roslyn it's using. For GitHub branches, SharpLab tells me the precise commit ID, which is great. 

But when using "Default" it doesn't say anything. I assume it's some recent version from NuGet, but it would be great if the UI said which version exactly.

**Comment by ashmind** [comment]
@svick Thanks, makes sense, done.

**Comment by xPaw** [comment]
Looks like this was removed at same point. Right now it just says "Default" without listing the actual version being used anywhere that I can see.

**Comment by ashmind** [comment]
Good point -- as far as I remember this got dropped when I changed to CDN-based static HTML from server-generated one.
It is possible to reimplement, but not as easy anymore.

It is possible to use `#error version` to get the version in the meantime, but I'll reopen this.
